src/main/java/org/retest/rebazer/connector/GithubConnector.java

```diff
@@ -63,11 +63,25 @@ public boolean isApproved( final PullRequest pullRequest ) {
 		final DocumentContext jsonPath = jsonPathForPath( requestPath( pullRequest ) + "/reviews" );
 		final List<String> reviews = jsonPath.<List<String>> read( "$..state" );
 		final String titleAndDescription = pullRequest.getTitle().concat( pullRequest.getDescription() );
+		safeReviewStates( pullRequest, reviews.size(), jsonPath );
 
 		return titleAndDescription.contains( "@All" ) && !pullRequest.getReviewers().isEmpty()
 				? pullRequest.getReviewers().values().stream().allMatch( "APPROVED"::equals )
 				: pullRequest.getReviewers().values().stream().anyMatch( "APPROVED"::equals );
 	}
+
+	private static void safeReviewStates( final PullRequest pullRequest, final long reviews,
```

Comment:
Inline parameter `reviews` because it is only used in this method.

Reply:
same for jsonPath.